sig-architecture/annual-report-2021.md

```diff
@@ -0,0 +1,102 @@
+  for those specialized purposes. Each of those programs has a documented shadow program to add new reviewers and approvers.
+- The main meeting itself is losing steam a bit. We could use more across the board attendance. We have to find ways to do this with better timely topics.
+- The sub projects are in sustainable shape for the most part. We could use more people in conformance, code-organization, and PRR for sure.
+- We want to rework the API reviews as well to make sure SIGs can for the most part do their own API reviews.
+- The Enhancements subproject has some tooling code, and is a good place for newer folks to contribute. It has been quite active recently, and as a process-focused subproject is a place where program managers can be 
+
+**What programs do you participate in for new contributors?**
+- Mainly through code-organization, we have a new mentee from LFX for example. 
+- We do need to grow more folks who can work across SIGs somehow. We might need to put together a mentoring program to help with this. One idea we had before of reading group for KEPs may need to be revived.
+
+**Does the group have contributors from multiple companies/affiliations? Can end users/companies contribute in some way that they currently are not?**
+
+- Yes. Our chairs are from three different companies. We have many participants and subproject owners from those same companies and others.
+
+## Current initiatives and project health
+
+- Please include links to KEPs and other supporting information that will be beneficial to multiple types of community members. 
+
+**What are initiatives that should be highlighted, lauded, shout out, that your group is proud of? Currently underway? What are some of the longer tail projects that your group is working on?**
+
+- We are proud of the work of the conformance subproject in burning down the [list of API endpoints that were untested](https://apisnoop.cncf.io/conformance-progress).
+- the code-organization has come a long way as well, helping coordinate updates to dependencies across various projects to help make our dependencies cleaner (example grpc update across etcd/containerd etc)
+- PRR ([policy](https://github.com/kubernetes/community/blob/master/sig-architecture/production-readiness.md), [KEP](https://github.com/kubernetes/enhancements/tree/master/keps/sig-architecture/1194-prod-readiness)) was made mandatory in 1.21 and the PRR team reviewed 66 KEPs, improving the scalability and supportability, and ensuring monitoring and feature enablement is properly conducted to make features production ready.
+- Policies to move features across the project to GA, including:
+    - [Conformance Without Beta](https://github.com/kubernetes/enhancements/tree/master/keps/sig-architecture/1333-conformance-without-beta)
+    - [Preventing Permabeta](https://github.com/kubernetes/enhancements/tree/master/keps/sig-architecture/1635-prevent-permabeta)
+
```

Comment:
We should add something about enhancements subproject here (kep improvements, receipts kep, process changes)

Reply:
Maybe something like:
- Process changes: The Enhancements team has been working with the RT to assist SIGs in taking greater ownership of their KEPs during the release cycle, reducing reliance on the RT and allowing them to better organize their work and increase communication between SIGs and authors.
- The Receipts Project (ongoing) is working on automating some of the mechanics of KEP process to streamline and better track KEPs in a release cycle.
- KEPs have been transitioned to the new format which includes kep.yamls and a new directory structure (special thanks to @wojtek-t and new contributor @shekhar-rajak for all of their hard work on this!).